bats/tests/k8s/up-downgrade-k8s.bats

```diff
@@ -3,13 +3,29 @@
 load '../helpers/load'
 ARCH_FOR_KUBERLR=amd64
 
+local_setup_file() {
+    if semver_eq "$RD_KUBERNETES_VERSION" "$RD_KUBERNETES_ALT_VERSION"; then
+        printf "Cannot upgrade from %s to %s\n" \
+            "$RD_KUBERNETES_VERSION" "$RD_KUBERNETES_ALT_VERSION" |
+            fail
+    fi
+    if semver_gt "$RD_KUBERNETES_VERSION" "$RD_KUBERNETES_ALT_VERSION"; then
```

Comment:
Maybe add a comment here _why_ we care that we upgrade and not downgrade?

Reply:
Will do. (But we actually do both upgrade and downgrade, we just expect a wipe for the downgrade.)